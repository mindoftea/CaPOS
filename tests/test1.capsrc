add 0x1234, 0x5678, 0x80000005
sub 0x80000005, 0xabc, 0x80000006
xor 0xff00, 0x80000006, 0x80000004
print 0x80000004, 0, 0
not 0x80000004, 0, 0x80000007
slr 0x80000007, 0x4, 0x8000000a
print 0x8000000a, 0, 0
