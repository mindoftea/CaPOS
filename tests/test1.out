0xa2f0
0xffff5d0
